juniper_official/Kernel/check-delayed-unrefs-anomaly-iagent.rule

```diff
@@ -0,0 +1,69 @@
+/*
+ * This rule checks if there are any delayed unrefs and if they are greater than
+ * a particular threshol (sysctl net.rt_nh_max_delayed_unrefs).
+ *
+ * Transient spikes in delayed unref count is expected in certain catastrophic or
+ * major events and is not a cause for worry. But, if this situation persists for
+ * significant amount of time (say in minutes), then it can result in traffic loss.
+ */
+iceberg {
+    topic kernel.ifstate {
+        rule check-delayed-unrefs-anomaly-iagent {
+            sensor delayed-unref-sensor {
+                iAgent {
+                    file delayed_unref.yml;
+                    table delayedunrefTable;
+                    frequency 30s;
+                }
+            }
+            field delayedcnt {
+                sensor delayed-unref-sensor {
+                    path unref_cnt;
+                }
+                type integer;
+            }
+            field delayedmax {
+                sensor delayed-unref-sensor {
+                    path unref_max;
+                }
+                type integer;
+            }
+            trigger report-delayed-unrefs-iagent {
```

Comment:
1) Provide trigger description and synopsis detailing sensor info.
2) Trigger frequency is must as default frequency is 10s and trigger gets evaluated once in 10s. but here sensor frequecy is 10 mins hence trigger frequency can be greaterthan equal to 10 mins.
e.g. synopsis "Routing engine GRES kpi";
       description "sets health based on Routing engine CPU utilization for 15 minutes";

Reply:
Addressed